tx/textual/valuerenderer/coins.go

```diff
@@ -76,7 +85,112 @@ func (vr coinsValueRenderer) Format(ctx context.Context, v protoreflect.Value) (
 	}
 }
 
-func (vr coinsValueRenderer) Parse(_ context.Context, screens []Screen) (protoreflect.Value, error) {
-	// ref: https://github.com/cosmos/cosmos-sdk/issues/13153
-	panic("implement me, see #13153")
+func (vr coinsValueRenderer) Parse(ctx context.Context, screens []Screen) (protoreflect.Value, error) {
+	if len(screens) != 1 {
+		return protoreflect.Value{}, fmt.Errorf("expected single screen: %v", screens)
+	}
+
+	if screens[0].Text == emptyCoins {
+		return protoreflect.ValueOfList(utils.NewGenericList([]*basev1beta1.Coin{})), nil
```

Comment:
This is incorrect. If the initial protobuf field was `Coin`, then we should return protoreflect.ValueOfMessage.

We can do:
```go
if vr.fd.IsList() {
  return ValueOfList
else {
  return ValueOfMessage
}
```